infrastructure/controller/pkg/endpoints/server.go

```diff
@@ -51,5 +51,8 @@ func Serve(clientSet *kubernetes.Clientset, namespace string) {
 	clusterUpdate := &ClusterUpdate{clientSet: clientSet, namespace: namespace}
 	r.Handle("/cluster.update", clusterUpdate)
 
+	enableDisableComponents := &EnableDisableComponents{clientSet: clientSet, namespace: namespace}
+	r.Handle("/components.enable", enableDisableComponents).Methods("POST")
+
```

Comment:
As the `config.Component` struct already has the `Enabled` parameter, perhaps we can also use the `component.update` endpoint? And then annotate the appropriate configMap. In that case, we will not need a new endpoint which perhaps makes the system simpler.